docs/guides-agent-app.md

```diff
@@ -0,0 +1,433 @@
+
+## 1. Install aragonCLI
+
+The [aragonCLI]((https://hack.aragon.org/docs/cli-intro).) (Command Line Interface) is what we use to create, interact with, and develop Aragon apps.
+
+Install it from NPM by running the following command:
+
+`npm install -g @aragon/cli`
+
+Hopefully, it downloaded successfully 😊.
+
+ If not, you should take a quick look at the [installing aragonCLI](https://hack.aragon.org/docs/guides-faq#installing-aragoncli) section of our [troubleshooting guide](/docs/guides-faq#installing-aragonCLI). It should help diagnose and fix the problem.
+ 
+If that still doesn't fix things 😟, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help). We're more than happy to help.
+
+Note that even if you've already installed the CLI, you might want to reinstall it to make sure you're up to date with the latest version.
+
+## 2. Install the Agent app
+
+Now that we've downloaded aragonCLI 🎉, we're ready to install the [Agent app](https://blog.aragon.one/aragon-agent-beta-release/).
+
+aragonCLI installs the [`aragon dao`](https://hack.aragon.org/docs/cli-dao-commands) commands. We use these cto interact directly with our DAO from the command line. They're also available directly using the `dao` shortcut.
+
+We'll use the the [`dao install`](https://hack.aragon.org/docs/cli-dao-commands#dao-install) command to install the Agent app.
+
+`dao install` takes two arguments:
+
+1. The address or name of an Aragon DAO. 
+2. The name of an Aragon app.
+
+So in our case, to install the Agent app, we'll run: 
+
+```
+dao install <your organisation name> agent --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+You should see that after `dao install <your organisation's name> agent` we pass in two [global options](https://hack.aragon.org/docs/cli-intro#global-options): `--enviroment` and `--apm.ipfs.rpc`.
+
+The `--environment` option allows us to specify the network we want to use. In our case we've created our organization on rinkeby so we pass in `aragon:rinkeby`.
+
+Note that if we had chosen the **Ethereum Mainnet** as the network for our organization we would have passed `aragon:mainnet` instead of `aragon:rinkeby` as the argument to `--environment`.
+
+The `--apm.ipfs.rpc` option allows us to point to an IPFS node that has the files we are looking for. In our case we're pointing it to the aragon network IPFS node.
+
+We can also run our own IPFS node by running `aragon ipfs` in another terminal window. 
+
+Running a local IPFS node allows us to run the same command without the `--apm.ipfs.rpc` option (since the `--apm.ipfs.rpc` option defaults to `http://localhost:5001`). 
+
+However, since IPFS propogation is slow, it's better to point directly to the aragon IPFS node.
+
+**Note that, in the true spirit of democracy, this step will trigger a vote in the DAO, you'll need to vote *yes* to confirm the installation of the Agent app.**
```

Comment:
```suggestion
**Note that this step will trigger a vote in the DAO, you'll need to vote _yes_ to confirm the installation of the Agent app. This is a consequence of how democracy governance of the organization was set up.**
```

Reply:
Clearly not as funny as I thought... 😋

Reply:
We should include the `in the true spirit of democracy` I think it was cause I worked in a previous version of your branch